cgt_calc/parsers/schwab_equity_award_json.py

```diff
@@ -199,6 +202,7 @@ def __init__(self, row: JsonRowType, file: str, field_names: FieldNames) -> None
         quantity = _decimal_from_number_or_str(row, names.quantity)
         amount = _decimal_from_number_or_str(row, names.amount)
         fees = _decimal_from_number_or_str(row, names.fees)
+        currency = "USD"
```

Comment:
Why have you moved it?

Reply:
I wanted to set `currency` to `None`, but then realised that it's a (non-optional) `str` in `BrokerTransaction` (https://github.com/KapJI/capital-gains-calculator/blob/main/cgt_calc/model.py#L83), so I reverted that change and forgot to undo this change to where the default is specified. I'm happy to undo this move.

Reply:
Yeah, let's undo.

Reply:
done, or well, undone.